client/src/modules/registration/registration.view.js

```diff
@@ -49,54 +98,54 @@ export default class Registration extends React.Component {
                 <ErrorCollector />
                 <div className="panel panel-default">
                     <div className="panel-heading">
-                        <h3 className="panel-title">Register new person</h3>
+                        <h3 className="panel-title"><Translate content="registerNewPerson" /></h3>
                     </div>
                     <div className="panel-body">
                         <div className="row">
                             <div className="col-md-12 col-sm-12 col-xs-12">
                                 <form role="form">
                                     <div className="form-group">
-                                        <label htmlFor="emial">Email</label>
+                                        <label htmlFor="email"><Translate content="email" /></label>
                                         <input type="email"
                                                className="form-control"
                                                id="email"
-                                               placeholder="Enter email"
+                                               placeholder={additionalConstants[counterpart.getLocale()].enterEmail}
```

Comment:
it will be better if you will create some method for additional constants which will return locale value which depends on current locale